samples/azure/self-host/Core_6/HostWorker/SafeRoleEnvironment.cs

```diff
@@ -0,0 +1,177 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+[DebuggerNonUserCode]
+static class SafeRoleEnvironment
+{
+    static bool isAvailable = true;
+    static Type roleEnvironmentType;
+    static Type roleInstanceType;
+    static Type roleType;
+    static Type localResourceType;
+
+    static SafeRoleEnvironment()
+    {
+        try
+        {
+            TryLoadRoleEnvironment();
+        }
+        catch
+        {
+            isAvailable = false;
+        }
+
+    }
+
+    public static bool IsAvailable => isAvailable;
+
+    public static string CurrentRoleInstanceId
+    {
+        get
+        {
+            if (!IsAvailable) throw new Exception("Role environment is not available, please check IsAvailable before calling this property!");
```

Comment:
add `{ }`

Reply:
`Role environment is not available, please check IsAvailable before calling this property!`

to 

`Role environment is not available, check IsAvailable before calling this property`